main/appserver/packager/appserver-base/src/main/docs/README.txt

```diff
@@ -1,4 +1,4 @@
-Thank you for downloading GlassFish Server Open Source Edition 4.1!
```

Comment:
Some of the other references to 4.1 in the same file which can be changed to 5.0